src/lib.rs

```diff
@@ -329,6 +329,16 @@ impl Config {
     /// otherwise cargo will link against the specified library.
     ///
     /// The given library name must not contain the `lib` prefix.
+    ///
+    /// # Example
+    ///
+    /// ```no_run
+    /// gcc::Config::new()
+    ///             .file("src/foo.c")
+    ///             .shared_flag(true)
+    ///             .cpp_set_stdlib(Some("stdc++"))
```

Comment:
I would expect the example for `cpp_link_stdlib` to call `cpp_link_stdlib`, not `cpp_set_stdlib`.